scalajslib/src/ScalaJSModule.scala

```diff
@@ -197,7 +197,7 @@ trait ScalaJSModule extends scalalib.ScalaModule { outer =>
 
   def moduleKind: T[ModuleKind] = T { ModuleKind.NoModule }
 
-  def useECMAScript2015: T[Boolean] = false
+  def useECMAScript2015: T[Boolean] = true
```

Comment:
As suggested in issue #1665, enable ECMAScript2015 only for ScalaJS 1.x.

```suggestion
  def useECMAScript2015: T[Boolean] = T { 
    !scalaJSVersion().startsWith("0.")
  }
```

Reply:
Looks good - thanks!